src/CareerBoostAI.Domain/Candidate/CvEntity/ValueObjects/CvId.cs

```diff
@@ -0,0 +1,30 @@
+﻿using CareerBoostAI.Domain.Common.Exceptions;
+using CareerBoostAI.Domain.ValueObjects;
+
+namespace CareerBoostAI.Domain.Candidate.Cv.ValueObjects;
+
+public class CvId : ValueObject
```

Comment:
This seems over-engineered, what benefit do we get instead of just instantiating a new CV object and setting its values?